mathics/builtin/list/associations.py

```diff
@@ -308,6 +308,13 @@ class Lookup(Builtin):
 
 
 class Missing(Builtin):
+    """
+    <dl>
+    <dd>'Missing[]'
+    <dt> represents a data that is misssing.
+    </dl>
+    """
+
     summary_text = "default value if a key is not found"
```

Comment:
This message is a little weird. Where did "key" come into this? 

https://reference.wolfram.com/language/guide/ImportingAndExporting.html says "symbolic representation of missing data". That makes sense to me. 

And under https://reference.wolfram.com/language/ref/Missing.html a nice example is given 

```
ElementData[117, "MeltingPoint"]
```

but I think a clearer example would be: 

```
ElementData["Meitnerium", "MeltingPOint"]
```

(I am a fan of Meitnerium)